src/guro/core/network.py

```
@@ -0,0 +1,421 @@
+    stats = {'tcp': {}, 'udp': {}}
+    if _SYSTEM != 'Linux':
+        return stats
+    try:
+        with open('/proc/net/snmp') as f:
+            lines = f.readlines()
+        for i, line in enumerate(lines):
+            parts = line.strip().split()
+            if parts[0] == 'Tcp:' and i + 1 < len(lines):
+                headers = parts[1:]
+                values = lines[i + 1].strip().split()[1:]
+                for h, v in zip(headers, values):
```

Ruff (0.15.15)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)